drivers/storage/portworx/deployment.go

```diff
@@ -802,6 +802,24 @@ func (t *template) getArguments() []string {
 		"-x", "kubernetes",
 	}
 
+	if t.cluster.Spec.Resources != nil && t.cluster.Spec.Resources.Limits != nil {
+		if memory, ok := t.cluster.Spec.Resources.Limits[v1.ResourceMemory]; ok {
+			bytes, b := memory.AsInt64()
+			if b {
+				args = append(args, "--memory")
+				args = append(args, fmt.Sprint(bytes))
+			} else {
+				logrus.Errorf("failed to parse memory %+v", memory)
+			}
+		}
+
+		if cpu, ok := t.cluster.Spec.Resources.Limits[v1.ResourceCPU]; ok {
+			f := cpu.AsApproximateFloat64()
```

Comment:
Does `AsApproximateFloat64` not return an "ok" value like the `AsInt64()` method?

Reply:
Yes